examples/pylab_examples/image_origin.py

```diff
@@ -1,25 +1,25 @@
-#!/usr/bin/env python
 """
 You can specify whether images should be plotted with the array origin
 x[0,0] in the upper left or upper right by using the origin parameter.
 You can also control the default be setting image.origin in your
 matplotlibrc file; see http://matplotlib.org/matplotlibrc
 """
-from pylab import *
+import matplotlib.pyplot as plt
+import numpy as np
 
-x = arange(100.0)
+x = np.arange(100.0)
 x.shape = (10, 10)
 
 interp = 'bilinear'
-#interp = 'nearest'
+# interp = 'nearest'
```

Comment:
Besides, if your linter is saying that that removing that space is a
violation of PEP8, then it is broken. Any linter that can not recognize
commented-out code is not worth using (in *any* language).

On Tue, Aug 18, 2015 at 8:14 PM, Eric Ma <notifications@github.com> wrote:

> In examples/pylab_examples/image_origin.py
> <https://github.com/matplotlib/matplotlib/pull/4950#discussion_r37368671>:
>
> >  x.shape = (10, 10)
> >
> >  interp = 'bilinear'
> > -#interp = 'nearest'
> > +# interp = 'nearest'
>
> Ok, got it. Removing...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4950/files#r37368671>.
>